src/test/java/Homework17.java

```diff
@@ -0,0 +1,86 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+import java.time.Duration;
+
+public class Homework17 extends LoginTests {
+
+    @Test
+    public void addSongToPlaylist() {
+
+        String playlistname = "test";
+        loginValidAccount("Test321@gmail.com", "te$t$tudent");
+        //deleteAddedSongPlaylist(playlistname);
+        //clickCreatePlaylist();
+        //inputPlaylist();
+        //enterPress();
+        //openHomeURL();
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._